### PR TITLE
config: Remove Coprocessor Cache config and disable it by default

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <me@breeswish.org>

### What problem does this PR solve? 

This PR removes the cache config so that user will not see them. Additionally this PR disables the cache by default becuase it has correctness issue now.

Correctness issue:
- Wall time 1: Write A with TS=10
- Wall time 2: Read A with TS=5 (read nothing, cached)
- Wall time 3: Read A with TS=15 (read nothing from cache, but should read A)

### What is changed and how it works?

Only config is changed. This PR does not solve the correctness issue.

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -554,10 +554,16 @@ var defaultConf = Config{
...
 
 		CoprCache: CoprocessorCache{
-			Enabled:               true,
-			CapacityMB:            1000,
-			AdmissionMaxResultMB:  10,
-			AdmissionMinProcessMs: 5,
+			//Enabled:               false,
+			//CapacityMB:            1000,
+			//AdmissionMaxResultMB:  10,
+			//AdmissionMinProcessMs: 5,
```
Comment (sticnarf) [inline config/config.go]: What are these comments for?

Comment (breeswish) [reply config/config.go]: Some recommended configurations if user want it back.

Comment (MyonKeminta) [reply config/config.go]: Did  you mean `Enabled:  true`?

File: config/config.go [hunk 2]
```diff
@@ -554,10 +554,18 @@ var defaultConf = Config{
 		StoreLimit:     0,
 
 		CoprCache: CoprocessorCache{
```
Comment (zz-jason) [inline config/config.go]: usually it's common that users shorten `coprocessor` to `cop`, I think `cop` might be a good name for users.

Comment (breeswish) [reply config/config.go]: Good idea. We can use this name when we re-introduce it later.

**Pull request merged**